infrateam_first_responder.md

```diff
@@ -66,14 +66,23 @@ Note that you will need to be sure you can ssh into each of the VMs from whereve
     - take note in #dlss-infra-stage-use if there is active testing going on;  be sure to either comment out that app or coordinate with tester
 - prod: if all tests passed for stage deploys, deploy to prod with script.
 
+###### Important Exceptions
+
+There are applications that need to be deployed separately (i.e., not using `sdr-deploy`):
+
+* **hydra_etd `uat` environment**: deploy via `cap uat deploy` in `hydra-etd`
+* **sul-pub environments beyond `stage` and `prod`**: deploy via `cap ENV deploy` ([ENV values](https://github.com/sul-dlss/sul_pub/tree/master/config/deploy))
+* **Sinopia apps**: deploy via terraform (see [DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/sinopia/operations-concerns.md#deployment-info))
+* **dlme-transform**: deploy via terraform (see [DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/dlme/operations-concerns.md#deployment-info))
+
 ##### Code that isn't a Ruby Application
 
-We have codebases to maintain that aren't Ruby applications or gems. We have not yet settled on a long-term method for dealing with these:
+We have codebases that aren't Ruby applications or gems. We have not yet settled on a long-term method for dealing with these:
 
-- javascript apps with npm updates:
-  - https://github.com/sul-dlss/access-update-scripts has code to create PRs for npm package updates, but it currently only works for the sul-dlss github organization. All the sinopia code is in the LD4P github organization. There is a [pending pull request](https://github.com/sul-dlss/access-update-scripts/pull/82) that will address this.
-- java code (note that we are down to two Java repositories in our platform: [suri2](https://github.com/sul-dlss/suri2) (which is slated to be replaced with the Rails-based [suri_rails](https://github.com/sul-dlss/suri_rails) in 2020), and [etd-reporter](https://github.com/sul-dlss/etd-reporter).)
+- java code
+  - the Infrastructure team is down to one Java repository in our platform, [etdpdf](https://github.com/sul-dlss/etdpdf), and it does not go through CI or receive maintenance updates
```

Comment:
Afraid not. https://github.com/sul-dlss/wasapi-downloader, https://github.com/sul-dlss/openwayback, and https://github.com/sul-dlss/WASMetadataExtractor.

Reply:
doh! thanks, @justinlittman. I've updated the document. Take another look?